cmd/libsnap-confine-private/mount-opt-test.c

```diff
@@ -275,6 +275,20 @@ static void test_sc_do_umount(gconstpointer snap_debug)
 	}
 }
 
+static bool missing_mount(struct sc_fault_state *state, void *ptr)
+{
+	errno = ENOENT;
+	return true;
+}
+
+static void test_sc_do_optional_mount(void)
+{
+	sc_break("mount", missing_mount);
+	bool ok = sc_do_optional_mount("/foo", "/bar", "ext4", MS_RDONLY, NULL);
+	g_assert_false(ok);
+	sc_reset_faults();
```

Comment:
This is obviously fine and tests the one conditional where optional is true and we get ENOENT, but I can't help but want for future-proofing for test_sc_do_optional_mount() to have all the tests as sc_do_mount(), but with optional as true. Not a blocker if this doesn't bother you the way it is nagging at me. :)

Reply:
Done :-)